src/app/game/troops/troops.component.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { TroopsComponent } from './troops.component';
+import { TroopsService } from './troops.service';
+import { HttpClientModule } from '@angular/common/http';
+
+
+describe('TroopsComponent', () => {
+  let component: TroopsComponent;
+  let fixture: ComponentFixture<TroopsComponent>;
+  let service: TroopsService;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [TroopsComponent],
+      imports: [HttpClientModule],
+      providers: [TroopsService]
```

Comment:
maybe mock the `TroopService`, not provide it

Reply:
changed to mock the service